src/az_ulib_ipc/az_ulib_ipc.c

```diff
@@ -485,3 +574,130 @@ AZ_NODISCARD az_result az_ulib_ipc_call_w_str(
 
   return result;
 }
+
+static az_result report_interfaces(uint16_t start, az_span* result, uint16_t* next)
+{
+  char* result_str = (char*)az_span_ptr(*result);
+  int32_t result_size = az_span_size(*result);
+  int32_t pos = 0;
+  uint16_t slot;
+
+  az_result res = AZ_ULIB_EOF;
+  for (slot = start; slot < AZ_ULIB_CONFIG_MAX_IPC_INTERFACE; slot++)
```

Comment:
nit: can you change this to 'i'? I see the value in slot, but 'i' is standard and I think it makes it slightly more readable

Reply:
I don't like `i` for variables that will transcend the `for` loop. But I agree that the name `slot` is not good, I think `interface_index` is a better name.